server/lobbyconnection.py

```diff
@@ -693,6 +693,7 @@ async def on_player_login(
         await self.send({
             "command": "welcome",
             "me": self.player.to_dict(),
+            "current_time": datetime.utcnow().timestamp(),
```

Comment:
Sure. There are several `now = datetime.utcnow()` [lines](https://github.com/baterflyrity/server/blob/eea6c98ae1b420046a25bb78d8a424c5e083af5a/server/lobbyconnection.py#L458) hence used that style.

Thus maybe it would be nice to migrate these UTCs to *server.timing* too.